packages/db/db-client-types.ts

```diff
@@ -248,33 +248,37 @@ export type ListUploadsOptions = {
 // Pinninng
 
 // PinRequest
-export type PAPinRequestUpsertInput = {
+export type PsaPinRequestUpsertInput = {
   id?: string,
-  name?: definitions['pa_pin_request']['name'],
+  name?: definitions['psa_pin_request']['name'],
   authKey: string,
-  requestedCid: definitions['pa_pin_request']['requested_cid'],
-  cid: definitions['pa_pin_request']['requested_cid'],
+  sourceCid: definitions['psa_pin_request']['source_cid'],
+  contentCid: definitions['upload']['content_cid'],
+  dagSize?: definitions['content']['dag_size'],
```

Comment:
Will we be able to get this at the PinRequest creation timing? From the API Spec it seems like we won't? https://ipfs.github.io/pinning-services-api-spec/#tag/pins/paths/~1pins/post

Reply:
I agree, I don't think we will but, honestly, we haven't dug into it just yet.
We have an [issue](https://github.com/web3-storage/web3.storage/issues/793) to look into it more closely.
Can I reserve to give you a more accurate answer when we start working on it?

Reply:
No we won't have this and can't get it from a worker. It will have to come from dagcargo after Cluster receives the data.